source/aggregation-tutorials.txt

```diff
@@ -0,0 +1,112 @@
+.. _pymongo-aggregation-tutorials-landing:
```

Comment:
This needs to be added to the `toc_landing_pages` array in snooty.toml to make it clickable in the toc